patch_test.go

```diff
@@ -90,6 +90,16 @@ var Cases = []Case{
 		`[ { "op": "remove", "path": "/foo/1" } ]`,
 		`{ "foo": [ "bar", "baz" ] }`,
 	},
+	{
+		`{ "foo": [ "bar", "qux", "baz" ] }`,
+		`[ { "op": "remove", "path": "/foo/-" , "value": "qux" }]`,
+		`{ "foo": [ "bar", "baz" ] }`,
+	},
+	{
+		`{ "foo": [ "3", "2", "1" ] }`,
+		`[ { "op": "remove", "path": "/foo/-" , "value": "1" }]`,
+		`{ "foo": [ "3", "2" ] }`,
+	},
```

Comment:
Could you add a test where the values in the array are objects? I'd like to see if that works.

Reply:
nicely!